docs/UserGuide.md

```diff
@@ -52,9 +52,9 @@ If you can type fast, TA<sup>2</sup> can get your contact and assignment managem
  * Result Message Box: Displays a success message if your command has been executed, or an error message if your command 
    has failed.
    
- * Contact List: Displays your student's relevant information. (Can be modified using `find`)
+ * Contact List Panel: Displays your student's relevant information. (Can be modified using `find`)
    
- * Assignment List: Displays a specified student's assignments. (Activated using `show`)
+ * Assignment List Panel: Displays a specified student's assignments. (Activated using `show`)
```

Comment:
Same for "Acitvated using show". I don't think the users understand what activated means, nor about what show is.